aspnet/web-api/overview/older-versions/using-web-api-1-with-entity-framework-5/using-web-api-with-entity-framework-part-1.md

```diff
@@ -1,8 +1,8 @@
 ---
 uid: web-api/overview/older-versions/using-web-api-1-with-entity-framework-5/using-web-api-with-entity-framework-part-1
 title: "Part 1: Overview and Creating the Project | Microsoft Docs"
-author: MikeWasson
-description: ""
+author: Rick-Anderson
+description: "In this tutorial, you will create a simple store web application using Entity Framework ASP.NET Web API."
```

Comment:
suggestion using active voice: Learn how to create a simple store web application using Entity Framework ASP.NET Web API.